decidim-admin/config/locales/en.yml

```diff
@@ -369,10 +374,19 @@ en:
           success: Participatory process step updated successfully.
       participatory_process_user_roles:
         create:
-          error: There was an error adding an admin for this participatory process.
-          success: Admin created successfully for this participatory process.
+          error: There was an error adding an admin role for this participatory process.
```

Comment:
The fact that this is a role is something internal, users are creating participatory process admins, not roles.